### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by biapar**
Change Google library import.

### CODE REVIEW

File: Reference/Security/Two-factor-authentication/index.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ namespace Umbraco9
             var member = _memberService.GetByKey(userOrMemberKey);
 
             var twoFactorAuthenticator = new TwoFactorAuthenticator();
-            SetupCode setupInfo = twoFactorAuthenticator.GenerateSetupCode("My application name", member.Username, secret, false);
+            SetupCode setupInfo = twoFactorAuthenticator.GenerateSetupCode("My application name", secret, 200,200  );
```
Comment (sofietoft) [inline Reference/Security/Two-factor-authentication/index.md:86]: What is the reason for removing the "member.Username"?
What does "200,200" mean? Would be great with a code-comment here, to explain the numbers 💪 

Also, if `member` should not be used, can line 83 be omitted? 🤔

**Review by sofietoft — COMMENTED** [review]
Thanks for the PR @biapar ! 💪 

I've added a comment in the code.
I'll make sure we test this as soon as possible 💪

File: Reference/Security/Two-factor-authentication/index.md (hunk 1 above)
Comment (biapar) [reply Reference/Security/Two-factor-authentication/index.md:86]: Hi, I use another library. In your example, you use this https://github.com/BrandonPotter/GoogleAuthenticator. I see that the version 3 support .NET Core 6.
Sorry for my mistake. You can reject the changes.

Comment (sofietoft) [reply Reference/Security/Two-factor-authentication/index.md:86]: Ah, I see!
So just to make sure I understand completely - is it the entire PR is can disregard, or only this part? 🤔

**Comment by sofietoft** [comment]
Hi @biapar !

I haven't heard back from you on this one, so I've decide to close it.
Since we've migrated the docs to a new platform, we wouldn't be able to merge this one in directly anyways.

Please feel free to open a new PR, if you still feel the article needs to be updated 💪

**Pull request closed without merging**